curly.go

```diff
@@ -72,6 +72,15 @@ func (c CurlyRouter) matchesRouteByPathTokens(routeTokens, requestTokens []strin
 			return false, 0, 0
 		}
 		requestToken := requestTokens[i]
+		if hasCustomVerb(routeToken){
```

Comment:
the hasCustomVerb is doing a regexp operation. this will affect performance. for this reason, I would like to introduce a field in the Router called   CustomVerbEnabled with an default value of false.

Reply:
now it will calculate a route has a custom verb when build route

Reply:
so, we don't need the CustomVerbEnabled !